spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ReactiveHealthEndpointWebExtensionUtil.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring.actuate;
+
+import static com.linecorp.armeria.spring.actuate.WebOperationService.handleResult;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.reactivestreams.Publisher;
+import org.springframework.boot.actuate.endpoint.InvalidEndpointRequestException;
+
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+final class ReactiveHealthEndpointWebExtensionUtil {
+
+    static HttpResponse handleMaybeReactorResult(SimpleHttpCodeStatusMapper statusMapper,
+                                                 ServiceRequestContext ctx, Object result,
+                                                 HttpMethod method) throws Throwable {
+        if (result instanceof Flux) {
```

Comment:
nice catch 👍

Reply:
Actually, I have a question.

If I understand correctly, `Flux` isn't actually returned anywhere yet.
https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/ReactiveHealthEndpointWebExtension.java

I'm wondering which is correct behavior when handling flux input:
1) Just return the first result and ignore the rest (like the default implementation of `Mono.from(Flux)`)
2) Collect all elements like done in the current PR

Reply:
> Flux isn't actually returned anywhere yet.

I couldn't find that either. 😅 

> I'm wondering which is correct behavior when handling flux input:

They just returned the collected list as a body with OK status so I followed it.
https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/reactive/AbstractWebFluxEndpointHandlerMapping.java#L385-L387
https://github.com/line/armeria/blob/master/spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/WebOperationService.java#L223-L225

Reply:
Understood. Fair enough 👍